scripts/bootstrap

```diff
@@ -66,17 +66,13 @@ virtualenv_path=`pipenv --venv`
 pipenv run npm i npm@latest -g --prefix "${virtualenv_path}" && pipenv run npm install --prefix "${virtualenv_path}" --silent -g node-sass@4.9.0 clean-css@3.4.19 uglify-js@2.7.3 requirejs@2.2.0 @angular/cli@7.0.4
 
 # collect static files and compile html/css assets
-CWD=`pwd`
 # build the angular ui application
 cd ui; pipenv run npm ci; pipenv run npm run deploy
 
 # install the npm dependencies
-cd ${CWD}
 pipenv run invenio npm
 static_folder=$(pipenv run invenio shell --no-term-title -c "print('static_folder:%s' % app.static_folder)"|grep static_folder| cut -d: -f2-)
-cd $static_folder
-pipenv run npm install
-cd ${CWD}
+pipenv run npm install $static_folder -g --prefix "${virtualenv_path}"
```

Comment:
-g seem not required

Reply:
I delete the `-g` option.
I understand the impact for a local OS, between all users and local user. But what about a pipenv environment? Does it have an impact in such environment?

Reply:
The -g make sense for command line tools such as angular node-sass etc. not for libraries.